book/security.rst

```diff
@@ -1352,6 +1352,15 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+.. note::
+
+    To enable the ``security:check`` command, you need to add the
+    `SensioDistributionBundle`_ in your composer dependencies.
```

Comment:
And I would remove "composer" here (if it was kept, you would have to uppercase it).